AdvancedC#/FileSystemVisitorPL/FileSystemVisitor.cs

```diff
@@ -1,7 +1,33 @@
-﻿namespace FileSystemVisitorPL
+namespace FileSystemVisitorPL
 {
+    public class FileSystemEventArgs : EventArgs
```

Comment:
Just for the future: it's a good practice to store all files in separate .cs files since it avoid making one file to overloaded